mautrix_instagram/portal.py

```diff
@@ -1192,6 +1191,10 @@ async def _convert_instagram_media_share(
                 }
                 combined["formatted_body"] = combined_formatted_body
 
+            if share_item.caption and item_type_name != "clip":
```

Comment:
why restrict by `item_type_name`? I feel like we should just include it all the time, even if it doesn't get used by the clients